1906-10-22.xml

```diff
@@ -0,0 +1,1678 @@
+                                <placeName>Liverpool</placeName> (via Bona) on <date
+                                when="1905-07-07">Friday, the 7th inst.</date> at 4 p.m.</p>
+                        <p>CARGO taken by special agreement only. Through Freights quoted for the
+                            UNITED STATES and INLAND TOWNS in GREAT BRITAIN.</p>
+                        <p>For passage or freight apply to the Agents, BARKER &amp; Co., Alexandria.
+                                <measure type="indexNo">2061-17-10-905</measure></p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-ghc01">
+                        <head>GELLATLY, HANKEY &amp; CO.</head>
+                        <p>SHIPOWNERS OF LONDON.</p>
+                        <p>Branches at Port Sudan, Suakin &amp; Jeddah</p>
+                        <p>Merchandise, furniture, baggage and personal effects forwarded, and
+                            insurances effected to all parts of the world.</p>
+                        <p>
+                            <measure type="indexNo">27376-28-2-907</measure>
+                        </p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-tcs03">
+                        <head>Thos. Cook &amp; Son, (EGYPT), LTD</head>
+                        <p>HEAD OFFICE: LUDGATE CIRCUS—LONDON.</p>
+                        <p>CHIEF EGYPTIAN OFFICE: — CAIRO, near SHEPHEARD'S HOTEL.</p>
+                        <p>Alexandria, Port Said, Suez, Luxor, Assuan, Halfa, and Khartum.</p>
+                        <p>GENERAL RAILWAY AND STEAMSHIP AGENTS. BANKERS.</p>
+                        <p>BAGGAGE AND FORWARDING AGENTS.</p>
+                        <p>Officially appointed &amp; Sole Agents in Cairo to the P.&amp;O. S.N.
+                            Co.</p>
+                        <p>Large and splendidly appointed steamers belonging to the Co. leave Cairo
+                            thrice weekly, between November and March, for Luxor, Assouan and Halfa
+                            in connection with trains de luxe to Khartoum.-Moderate fares.</p>
+                        <p>Specially Reduced Rates for residents of Egypt by Tourist Services during
+                            November and December. First sailing November 13th.</p>
+                        <p>WEEKLY FREIGHT SERVICE FROM CAIRO TO ASSUAN AND HALFA.</p>
+                        <p>Cook's Interpreters in uniform are present at principal Railway Stations
+                            and Landing-places in Europe to assist passengers holding their
+                            tickets.</p>
+                        <p>Special Steamers and Dahabeahs for private parties.</p>
+                        <p>Special arrangements for tour in PALESTINE, SYRIA and the DESERT; Lowest
+                            rates.</p>
+                        <p>Best camp equipment in the country.</p>
+                    </div>
+
+                </div>
```

Comment:
These two `</div>` tags, which close the section and the page, should come at the end of all of the ads. The best way to do this is to move all of the ads you've put into page 2 and page 3 (below) onto the line just before these tags.